manifests/aws.pp

```diff
@@ -0,0 +1,41 @@
+# Manifest to configure the pre-requisites for puppetlabs-provision module
+#
+# @param access_key_id
+#    Optional IAM access key with EC2 privileges
+#
+# @param secret_access_key
+#    Optional IAM secret access key associated with the access key ID
+#
+# @param session_token
+#    Optional session token for AWS STS credentials, as the STS credentials are bound with time so this 
+#    parameter is not supported with Hiera configurations.
+#
+# @param profile
+#    Optional profile name to use for credentials
+#
+class provision::aws (
```

Comment:
maybe call this aws_credentials or aws_config to make it clear what this class does?